src/webportal/src/app/job-submission/components/yamledit-topbar/export-config.jsx

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
add lisence header following https://docs.opensource.microsoft.com/content/releasing/copyright-headers.html?q=header

Reply:
Added in new commit